assets/arc-0052/contextual.api.crypto.ts

```diff
@@ -8,18 +8,12 @@ import {
     ready,
     crypto_sign_ed25519_pk_to_curve25519,
     crypto_scalarmult,
-    crypto_sign_detached,
-    crypto_sign,
-    crypto_sign_SECRETKEYBYTES,
-    crypto_sign_ed25519_sk_to_pk,
-    crypto_scalarmult_ed25519_base,
     crypto_generichash,
-    crypto_scalarmult_base
 } from 'libsodium-wrappers-sumo';
 import * as msgpack from "algo-msgpack-with-bigint"
 import Ajv from "ajv"
 import { deriveChildNodePrivate, fromSeed } from './bip32-ed25519';
-import { randomBytes } from 'crypto';
+import { Transaction, EncodedSignedTransaction } from 'algosdk';
```

Comment:
Let's not use algosdk Types. They couple the implementations together and it brings a lot more than just types, specially third-party dependencies.